fiftyone/server/routes/geo.py

```
@@ -28,49 +27,57 @@
 
     results = {}
 
-    async for result in agg_results:
-        if result and "sampleId" in result and "coordinates" in result:
-            results[result["sampleId"]] = result["coordinates"]
+    async for doc in collection.aggregate(pipeline):
+        try:
+            results[str(doc["_id"])] = doc["coordinates"]
+        except:
+            # invalid/missing entry
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)